### DISCUSSION TITLE
Unable to get Sepolia Eth for free!

### DISCUSSION THREAD

**Discussion by AnthonyHarvey8867** [Cyfrin/foundry-full-course-cu#222] [Q&A] [answered]
Hi guys, has something changed with Eth Faucets out there? is it just me or they don't provide you with free sepolia eth any longer?
can someone help me out in this regard?

**Comment by Shiva-Sai-ssb** [comment, marked as answer]
Hello @AnthonyHarvey8867, you can obtain SepoliaETH through the alchemy faucet without any cost. You can access it by visiting the following link: https://sepoliafaucet.com/

**Reply by AnthonyHarvey8867** [reply]
thanks. I had a problem creating an account on alchemy platform and it just kept getting me an error. that was why I tried to find another way to do so. I finally tried a couple of more emails to sing up and got it to work.
thanks for your reply

**Comment by 0xtdey** [comment]
You can use chainlink faucet or alchemy faucet to get SepoliaETH for free, but catch is you can only get a certain amount in 24 hours in a single account. Or if none of them works you can just mine it from here - https://sepolia-faucet.pk910.de/

**Comment by IAm0xAxe** [comment]
You can get SepoliaETH through this link "https://chaintool.tech/faucet"
